.github/workflows/ci.yml

```diff
@@ -113,3 +135,43 @@ jobs:
         run: |
           ./cc-test-reporter sum-coverage coverage/**/*.json
           ./cc-test-reporter upload-coverage
+  update-docs:
```

Comment:
instead of having this `update-docs` job, running the request specs for a second time, why not doing:
1. remove this `update-docs` job
2. in the `tests` job adding `echo "OPENAPI=$(github.event_name == 'push' && github.ref == 'refs/heads/main')" >> $GITHUB_ENV` so that OPENAPI=true only in `main`